defaults/main.yml

```diff
@@ -130,9 +137,9 @@ tor_exit_notice_file: tor-exit-notice.html
 #
 # Quad9 DNS IPs:
 #  9.9.9.9
-#  9.9.9.10
-#  149.112.112.10
```

Comment:
Can you elaborate on the reasoning behind this removal?
These IPs quad9 resolvers as per: https://www.quad9.com/faq/

Reply:
thank you for the correct url, I just followed the existing link to confirm IPs, fixed